### PR TITLE
GraphQL-741: Add extension point to set custom parameters to Query Context object

### PR DESCRIPTION
**Pull request by naydav**
#741

### CODE REVIEW

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5288](https://jira.corp.magento.com/browse/ENGCOM-5288) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?php
```
Comment (paliarush) [inline app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php:1]: Extension attributes must be declared in `extension_attributes.xml`

Comment (paliarush) [inline app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php:1]: Please document how to extend context object.

**Review by lenaorobei — APPROVED** [review]

File: app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php (hunk 1 above)
Comment (naydav) [reply app/code/Magento/CustomerGraphQl/Model/Context/AddUserInfoToContext.php:1]: Fixed

**Pull request merged**